packages/dialtone-vue2/components/link/link_constants.js

```diff
@@ -2,21 +2,36 @@ export const DANGER = 'danger';
 export const WARNING = 'warning';
 export const SUCCESS = 'success';
 export const MUTED = 'muted';
-export const INVERTED = 'inverted';
 export const MENTION = 'mention';
-export const LINK_VARIANTS = ['', DANGER, WARNING, SUCCESS, MUTED, INVERTED, MENTION];
+export const LINK_VARIANTS = ['', DANGER, WARNING, SUCCESS, MUTED, MENTION];
 
 export const LINK_KIND_MODIFIERS = {
   default: '',
   warning: 'd-link--warning',
   danger: 'd-link--danger',
   success: 'd-link--success',
   muted: 'd-link--muted',
-  inverted: 'd-link--inverted',
   mention: 'd-link--mention',
 };
 
+const LINK_KIND_MODIFIERS_INVERTED = {
+  '': 'd-link--inverted',
```

Comment:
Maybe we can use `default: 'd-link--inverted'` to be consistent with `LINK_KIND_MODIFIERS`